providers/infrastructure-aws/v2.0.2/yttcc/overlay.yaml

```diff
@@ -54,7 +54,7 @@ spec:
       metadata:
         annotations:
           #! VVV TODO(vui) os-name handling
```

Comment:
consider removing the TODO, now that this is resolved

Reply:
ok, removed